ApplicationLibCode/Application/Tools/RiaEnsembleNameTools.cpp

```diff
@@ -32,21 +33,117 @@
 #include <QFileInfo>
 #include <QRegularExpression>
 
-template <>
-void caf::AppEnum<RiaEnsembleNameTools::EnsembleGroupingMode>::setUp()
+#include <regex>
+
+namespace internal
+{
+auto normalizePath = []( const std::string& path )
```

Comment:
Should this be a method in RiaFilePathTools?

Reply:
Agree